### DISCUSSION TITLE
Service Partner Kommentar im Grid anzeigen

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#3] [Shipped - Live on production]
Service Partner Kommentar im Grid mit anzeigen, um schneller filtern und suchen zu können

<img width="1508" alt="Bildschirmfoto 2022-05-30 um 11 38 11" src="https://user-images.githubusercontent.com/80519295/170964545-d9126b34-d49d-4b3a-8f88-fd6cf49b84aa.png">

<img width="1508" alt="Bildschirmfoto 2022-05-30 um 11 37 19" src="https://user-images.githubusercontent.com/80519295/170964662-f256e1de-31f1-435d-abc3-c76acf1a4de5.png">
nen

**Comment by YvesR** [comment]
Model do not allow such a thing, also the comment field is bad for anything.
We probably replace this by categories we add to contacts
